main.go

```diff
@@ -81,6 +84,9 @@ func run(coverprofile string, args []string, covermode, cpu, parallel, timeout s
 		usage()
 		return nil
 	}
+	if concurrency < 1 {
+		return fmt.Errorf( "invalid concurrency value '%d', value must be at least 1\n", concurrency)
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline